Newsletters/_posts/Yearn-Finance-Newsletter-54/2022-01-20-Yearn-Finance-Newsletter-54.md

```diff
@@ -4,86 +4,86 @@ title: "Yearn Finance Newsletter #54"
 categories: [Newsletters]
 image: ./Yearn-Finance-Newsletter-54/image1.png
 author: Yearn
-translator:
+translator: Stathis
 publish: false
```

Comment:
```suggestion
publish: false
```
change to 
`publish: true`

Reply:
Pushed some changes including the above. Also added some changes for consistency reasons based on the comments of NL#55..